airbyte-integrations/connectors/source-recharge/source_recharge/api.py

```diff
@@ -35,7 +34,7 @@ def data_path(self):
         return self.name
 
     def path(
-        self, stream_state: Mapping[str, Any] = None, stream_slice: Mapping[str, Any] = None, next_page_token: Mapping[str, Any] = None
+            self, stream_state: Mapping[str, Any] = None, stream_slice: Mapping[str, Any] = None, next_page_token: Mapping[str, Any] = None
```

Comment:
These extra indentations don't look right. Are your pre-commit hooks not working right?

Reply:
Started formatting manually. Should be better with extra indentations.